experimental/traffic-portal/src/app/api/statuses.service.ts

```diff
@@ -0,0 +1,71 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { APIService } from './base-api.service';
+
+export interface StatusesModel {
+	description?: string;
+	id?: number;
+	lastUpdated?: Date;
+	name?: string;
+}
+
+@Injectable({
+	providedIn: 'root'
```

Comment:
New API services must not be provided in the root scope, they are to be provided by the API module. That way, it can be easily swapped out as the API Testing module has mock providers for all of them. Similar to how the Angular `RouterModule` and `RouterTestingModule` modules work.